pkg/scheduler/metrics/metrics.go

```diff
@@ -301,6 +308,38 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		}, []string{"type"})
 
+	ReadyTime = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Subsystem: SchedulerSubsystem,
+			Name:      "ready_time",
```

Comment:
What's the use case for this metric?

Reply:
We can get  the existed`process_start_time`, but the time doesn't mean leader scheduler is ready to work,  so I add the `scheduler_ready_time`.

Reply:
I think the `process_start_time` is used to reset counter metrics. The metric itself isn't that interesting to me, same as the `scheduler_ready_time`. Unless there is a specific use case.